modules/ting/js/ting.js

```diff
@@ -9,7 +9,7 @@
   Drupal.behaviors.ting = {
     attach: function (context) {
       // TODO: These selectors might be outdated.
-      $('a.js-search-overlay', context).live('click', function () {
+      $('a.js-search-overlay', context).on('click', function () {
```

Comment:
@guddo .live() is deprecated in jQuery 3.x, so if this functionality will be called it will fail.